modular-docs-manual/content/topics/creating_procedure_modules.adoc

```diff
@@ -18,29 +27,37 @@ The introduction typically provides context for the procedure, such as:
 * Why and where the user performs the procedure
 * Special considerations specific to the procedure
 
-.Procedure prerequisites
-This section is optional. Prerequisites are a bulleted list of conditions that must be satisfied before the user starts the procedure. Use a bulleted list for prerequisites and the plural heading *Prerequisites*, even if you only have a single prerequisite.
+.Limitations
+This section is optional and not used often. If there are any limitations that the affect the procedure, list them in this section.
```

Comment:
Do we care to tell them to use the plural here like we do on prereqs?

Reply:
Yes, changed.